app/models/group_assignment_invitation.rb

```diff
@@ -42,6 +43,11 @@ def enabled?
     group_assignment.invitations_enabled?
   end
 
+  def status(group)
+    group_invite_statuses.find_by(group: group) ||
```

Comment:
🎨 just a stylistic thing: might want to refactor this method to follow the structure of other methods in this file, something like this:

```
group_invite_status = group_invite_statuses.find_by(group: group)
return group_invite_status if group_invite_status.present?

GroupInviteStatus.create(group: group, group_assignment_invitation: self)
```

Reply:
This is styled like status method on `assignment_invitation.rb`
https://github.com/education/classroom/blob/9d38c3649d0d9c370f63ba1c464d23195a1044ea/app/models/assignment_invitation.rb#L62-L65

Should I re-style that too?

Reply:
ooh I see 🤔 imo, the second style is a bit easier to read even though it's more code, what do you think?

If you do refactor this method, it makes sense to refactor the one in `assignment_invitation.rb` too 👍